consensus/safety-rules/src/counters.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use safety_push_metrics::{define_counters, MetricsPusher, Gauge, Counter};
+use std::sync::Arc;
+
+// Use the libra_safety_rules prefix for all counters
+define_counters![
+    "libra_safety_rules",
+    (sign_proposal: Counter, "sign_proposal counter counts sign_proposals"),
+    (sign_timeout: Counter, "sign_timeout counter counts sign_timeouts"),
+    (some_gauge_counter: Gauge, "example help for a gauge metric"),
+];
+
+lazy_static! {
+    pub static ref COUNTERS: Arc<Counters> = Arc::new(Counters::new());
+    pub static ref METRICS_PUSHER: MetricsPusher = MetricsPusher::new(COUNTERS.clone());
```

Comment:
why do we want this thread to be a lazy static? is it because we don't have a single handler for safety rules?

Reply:
There is absolutely no need for lazy static. Removed it.